Ghidra/Features/FunctionID/ghidra_scripts/CreateEmptyFidDatabase.java

```diff
@@ -0,0 +1,45 @@
+/* ###
+ * IP: GHIDRA
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+// Create and add (attach) an empty FID database.
+// The functionality is similar to "Tools -> Function ID -> Create new empty FidDb..."
+// but this script can be executed in both GUI and headless modes.
+// The created database can later be populated,
+// for example by `CreateMultipleLibraries.java` when running in headless mode or
+// by "Tools -> Function ID -> Populate FidDb from programs..." when running in GUI mode.
+//@category FunctionID
+import ghidra.app.script.GhidraScript;
+import ghidra.feature.fid.db.FidFileManager;
+import java.io.File;
+
+public class CreateEmptyFidDatabase extends GhidraScript {
+
+	@Override
+	protected void run() throws Exception {
+		File dbFile;
+		// headless mode: `askFile()` cannot be used to specify the output file in headless mode
+		// because the file does not exist yet and `IllegalArgumentException` will be raised
+		if (getScriptArgs().length == 1) {
+			dbFile = new File(getScriptArgs()[0]);
+		}
+		// GUI mode
+		else {
+			dbFile = askFile("Create new FidDb file", "Create");
```

Comment:
I'm confused about `askFile`...if you do this in the GUI, how does the file get created in order to make `askFile()` happy?

Reply:
Nevermind, i stepped through to see what is happening.  This behavior seems weird...I'm going to bring it up to the team.

Reply:
Thank you. I did not really investigate it much beyond checking the [docs](https://github.com/NationalSecurityAgency/ghidra/blob/master/Ghidra/Features/Base/src/main/java/ghidra/app/script/GhidraScript.java#L2041) for `askFile` where I saw that it cannot be used in headless mode to create files. If you have any suggestions for replacing `getScriptArgs` or rewriting the logic let me know.

Reply:
Agreed

Reply:
I created #7025 to track this and #7026 with a possible simple fix. Please review it if you are interested in it.